libs/vm-modules/include/vm_modules/ml/model/model_estimator.hpp

```diff
@@ -179,6 +185,14 @@ class ModelEstimator
   {
     return DataType(0.003333333333333);
   };
+  static constexpr DataType SCEL_FORWARD_IMPACT()
+  {
+    return DataType(0.003333333333333);
```

Comment:
Konstantinos' PR is in, so FixedPoint initialisations should all be strings

Reply:
Jiri has done these in his PR